addons/animated-thumb/addon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Thumbnails Setter",
+  "description": "This addon will allow you to set the thubnail of your project to any image (including gif images)",
```

Comment:
1. Typo "thubnail"
2. "This addon will allow you to set the thumbnail of your project" how? You should mention where the button is placed.
3. Repetition of word "image"
4. Missing dot at the end